flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java

```diff
@@ -382,8 +384,6 @@ private FlexLinesResult calculateFlexLines(int mainMeasureSpec, int crossMeasure
 
         boolean reachedToIndex = toIndex == NO_POSITION;
 
-        boolean isMainHorizontal = mFlexContainer.isMainAxisDirectionHorizontal();
```

Comment:
I thought the most important info in this method is if the main axis horizontal so I moved the variable to the start of the method.